src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -103,6 +102,7 @@ function AppSettingsModal({
       isOpen
       onClose={onClose}
       size="md"
+      closeText={intl.formatMessage(messages.cancel)}
```

Comment:
@xitij2000 Just curious, where this came from? 
https://paragon-edx.netlify.app/components/modal/modal-dialog/

Reply:
@awaisdar001  I am not sure if this is what you asked for but the `intl` is provided from [intlInject in frontend-platform](https://github.com/edx/frontend-platform/blob/c15ebde3b3/src/i18n/injectIntlWithShim.jsx) if I am not wrong.

Reply:
sorry for not being clear - I was asking for `closeText` param.